renderer/pages/index.tsx

```diff
@@ -376,6 +388,11 @@ export default function Home() {
 
   return (
     <div className="h-full bg-gray-100 dark:bg-dark-back">
+      {!isDropboxConnected && (
+        <span className="text-yellow-600 inline-flex  items-center px-2.5 py-0.5 rounded-full text-xs font-medium bg-yellow-100 dark:text-yellow-400 dark:bg-yellow-400/20">
+          Dropbox is not connected
+        </span>
+      )}
```

Comment:
do we really need to display note when we don't use dropbox? As for me, it would be better to reverse this logic e.g. show only when we connected